spec/bns/fetcher/notion/pto_next_week_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+RSpec.describe Fetcher::Notion::PtoNextWeek do
+  before do
+    @config = {
+      database_id: "8187370982134ed099f9d14385aa81c9",
+      secret: "secret_K5UCqm27GvAscTlaGJmS2se4fyM1K7is3OIZMw03NaC"
+    }
+
+    @fetcher = described_class.new(@config)
+  end
+
+  describe "attributes and arguments" do
+    it { expect(@fetcher).to respond_to(:config) }
+
+    it { expect(described_class).to respond_to(:new).with(1).arguments }
+    it { expect(@fetcher).to respond_to(:fetch).with(0).arguments }
+  end
+
+  describe ".fetch" do
+    it "fetch data from the given configured notion database" do
```

Comment:
Perhaps for added clarity, it's worth considering including the test when using a non-empty filter, similar to the other pull request, what do you think? 🤔

Reply:
Same thing as the one in #41, I'm going to update the old test also and the name of the VCR.